test/js/components/tasks/cards.test.jsx

```diff
@@ -443,6 +442,166 @@ describe('<OrgCards/>', () => {
+            task: {
+              ...defaultTask,
+              commits: [],
+              pr_is_open: true,
+              review_valid: true,
+            },
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+              },
+            },
+          });
+
+          expect(getByText('Update Review')).toBeVisible();
+        });
+
+        test('disabled', () => {
+          const { getByText } = setup({
+            task: {
+              ...defaultTask,
+              commits: [],
+              pr_is_open: true,
+              review_valid: false,
+            },
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: false,
+              },
+            },
+          });
+          fireEvent.focus(getByText('Submit Review'));
+
+          expect(getByText('Submit Review')).toHaveAttribute('disabled');
+        });
+      });
+
+      describe('orgStatus', () => {
+        test('renders "Approved" status', () => {
+          const { getByText } = setup({
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+                url: 'url',
+              },
+            },
+            task: {
+              ...defaultTask,
+              commits: [],
+              review_status: 'Approved',
+              review_valid: true,
+            },
+          });
+          // debug();
+          expect(getByText('Approved')).toBeVisible();
+        });
+
+        test('renders "Changes requested" status', () => {
+          const { getByText } = setup({
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+                url: 'url',
+              },
+            },
+            task: {
+              ...defaultTask,
+              commits: [],
+              review_status: 'Changes requested',
+              review_valid: true,
+            },
+          });
+
+          expect(getByText('Changes requested')).toBeVisible();
+        });
+
+        test('renders "Review out of date" status', () => {
+          const { getByText } = setup({
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+                url: 'url',
+              },
+            },
+            task: {
+              ...defaultTask,
+              commits: [],
+              review_status: 'Approved',
+              review_valid: false,
+            },
+          });
+
+          expect(getByText('Review Out of date')).toBeVisible();
+        });
```

Comment:
@jgerigmeyer This test is failing.  what's the right util to asserts the display text, when it's broken up into different lines? :

```
Review out of date
| Submitted review 4 months ago
```

Reply:
Yep, what you're looking for is `{ exact: false }` -- https://testing-library.com/docs/dom-testing-library/api-queries#precision